src/commands/templates.rs

```diff
@@ -390,6 +390,10 @@ impl Uninstall {
 /// List the installed templates.
 #[derive(Parser, Debug)]
 pub struct List {
+    /// Filter templates with associated tags.
```

Comment:
May be worth saying if this filters to templates with _all_ given tags or _any_ given tag.  (I think it's the former and I think that's the right decision... just suggesting docs.)